### PR TITLE
Feature/add frontend function paddle tensor quantile

### LINKED ISSUE #22577: <quantile>
- [ ] #15115

### PR DESCRIPTION
**Pull request by AliTarekk**
# Add frontend function `paddle.tensor.tensor.Tensor.quantile`

## Related Issue 

Close #22577

### CODE REVIEW

**Comment by AliTarekk** [comment]
@Mr-Niraj-Kulkarni Kindly review my PR. Thanks!

**Comment by Mr-Niraj-Kulkarni** [comment]
@AliTarekk  Please resolve the lint errors, also I see the unit tests are failing please. Thanks

File: ivy_tests/test_ivy/test_frontends/test_paddle/test_tensor/test_tensor.py [hunk 1]
```diff
@@ -2908,3 +2908,38 @@ def test_paddle_tensor_trunc(
...
+    method_flags,
+    frontend,
+    backend_fw,
+    on_device,
+):
+    input_dtype, x = dtype_and_x
+    helpers.test_frontend_method(
+        init_input_dtypes=input_dtype,
+        init_all_as_kwargs_np={
+            "data": x[0],
```
Comment (Mr-Niraj-Kulkarni) [inline ivy_tests/test_ivy/test_frontends/test_paddle/test_tensor/test_tensor.py:2934]: please refer the implementation of test above for passing the function arguments

File: ivy_tests/test_ivy/test_frontends/test_paddle/test_tensor/test_tensor.py [hunk 2]
```diff
@@ -2908,3 +2908,38 @@ def test_paddle_tensor_trunc(
...
+        },
```
Comment (Mr-Niraj-Kulkarni) [inline ivy_tests/test_ivy/test_frontends/test_paddle/test_tensor/test_tensor.py:2935]: Also add other parameters which are missing q, axis, keepdims

File: ivy_tests/test_ivy/test_frontends/test_paddle/test_tensor/test_tensor.py (hunk 1 above)
Comment (AliTarekk) [reply ivy_tests/test_ivy/test_frontends/test_paddle/test_tensor/test_tensor.py:2934]: Please elaborate this point further, I don't really get what you mean. Thanks!

File: ivy_tests/test_ivy/test_frontends/test_paddle/test_tensor/test_tensor.py (hunk 2 above)
Comment (AliTarekk) [reply ivy_tests/test_ivy/test_frontends/test_paddle/test_tensor/test_tensor.py:2935]: Thanks for your review, Done!

**Comment by AliTarekk** [comment]
> @AliTarekk Please resolve the lint errors, also I see the unit tests are failing please. Thanks

Regarding the lint errors, I resolved them but the error persists for some reason. Thanks!

**Comment by AliTarekk** [comment]
@Mr-Niraj-Kulkarni Kindly re-review my PR. Thanks!

**Comment by ivy-seed** [comment]
This PR has been labelled as stale because it has been inactive for more than 7 days. If you would like to continue working on this PR, then please add another comment or this PR will be closed in 7 days.

**Comment by ivy-seed** [comment]
This PR has been closed because it has been marked as stale for more than 7 days with no activity.

**Pull request closed without merging**